shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -211,6 +239,7 @@ public Options getOptions() {
         .addOption(LIST_DIR_AS_FILE_OPTION)
         .addOption(LIST_HUMAN_READABLE_OPTION)
         .addOption(LIST_PINNED_FILES_OPTION)
+        .addOption(TIMESTAMP_OPTION)
```

Comment:
NIT: Keep the options sorted alphabetically.

Reply:
Sorted.